packages/esm-patient-forms-app/src/forms/forms-table.component.tsx

```diff
@@ -88,6 +106,21 @@ const FormsTable = ({ tableHeaders, tableRows, isTablet, handleSearch, handleFor
               </Table>
             )}
           </TableContainer>
+          <Pagination
+            forwardText={t('nextPage', 'Next page')}
```

Comment:
I'm noticing that the pagination container is getting cut off on the right:

![CleanShot 2025-06-25 at 17 17 51@2x](https://github.com/user-attachments/assets/4625f236-ac3b-4e65-81be-97752bb83e27)

An easy fix that doesn't require adjusting width values might be to set a different label for the Pagination container. For example:

```
itemsPerPageText={t('noOfForms', 'No. of forms')}
```

This ensures the component fits within the available horizontal space and has the added advantage of being clearer and more concise:

![CleanShot 2025-06-25 at 17 22 12@2x](https://github.com/user-attachments/assets/a7febe85-3991-403f-b118-6757a1de8672)